lucien-ios/ComicDetailScreen/ComicDetailScreenViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ComicDetailScreenViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/24/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+final class ComicDetailScreenViewController: UIViewController, UIScrollViewDelegate {
+
+    @IBOutlet private var scrollView: UIScrollView!
+    @IBOutlet private weak var comicFadeImageView: UIImageView!
+    @IBOutlet private weak var comicImageCover: UIImageView!
+    @IBOutlet private weak var seriesTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+
+    private var viewModel: ComicDetailViewModel
+    let disposeBag = DisposeBag()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
```

Comment:
1) remove this func if only calling super 2) the nav bar here doesn't match the designs